### PR TITLE
Upgrade CakePHP to version 3.7

### PR DESCRIPTION
**Pull request by oozone**
Upgrade CakePHP to version 3.7

### CODE REVIEW

File: frameworks/PHP/cakephp/src/Controller/FortunesController.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        $this->loadModel('Fortune');
+	$this->set('title_for_layout', 'Fortunes');
+
+        $fortunesTable = TableRegistry::getTableLocator()->get('Fortune');
+
+        $query = $fortunesTable->find('all');
+
+        // Calling all() will execute the query
+        // and return the result set.
+        $results = $query->all()->toArray();
```
Comment (joanhey) [inline frameworks/PHP/cakephp/src/Controller/FortunesController.php]: Please, use objects to have Full ORM.

> Using an ORM, all Fortune objects must be fetched from the Fortune table, and placed into a list data structure.

**Comment by joanhey** [comment]
It's not necessary to close the PR every time.

Please try to delete unnecessary files.

File: frameworks/PHP/cakephp/src/Controller/FortunesController.php (hunk 1 above)
Comment (oozone) [reply frameworks/PHP/cakephp/src/Controller/FortunesController.php]: Will do!

Comment (joanhey) [reply frameworks/PHP/cakephp/src/Controller/FortunesController.php]: Example for sorting the objects:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/PHP/yii2/app/controllers/SiteController.php#L52-L62

**Comment by joanhey** [comment]
Ready to be merged :+1:

**Comment by nbrady-techempower** [comment]
Thanks for reviewing @joanhey!

**Pull request merged**